src/pages/components/activity-graph/activity-data.js

```diff
@@ -0,0 +1,3224 @@
+const activityData = {
```

Comment:
My gut feeling is that it might be more efficient to make this a JSON file, since [JS can parse JSON faster than JS](https://v8.dev/blog/cost-of-javascript-2019#json).

Webpack should handle `.json` better than `.js` too, but I'm not 100% sure about that.

Not super important since it's just an optimisation thing.